### PR TITLE
120 create component add student popup

### PR DESCRIPTION
**Pull request by enx-code**

### CODE REVIEW

File: src/components/Cohorts/AddStudentPopUp/addStudentPopUp.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+.student-popup-ul {
+  border: 1px solid white;
+  border-radius: 10px;
+  padding: 20px;
+  list-style: none
+}
+.student-popup-list {
+  cursor: pointer;
+  line-height: 3;
```
Comment (lucianosimoni) [inline src/components/Cohorts/AddStudentPopUp/addStudentPopUp.css:45]: 3 what? Must be specified.

**Review by lucianosimoni — CHANGES_REQUESTED** [review]

**Review by lucianosimoni — APPROVED** [review]
🧙‍♂️

File: src/components/Cohorts/AddStudentPopUp/AddStudentPopUp.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  setRenderStudentsPopup,
+  students,
+  updateStudentsList
+}) {
+  const studentsWithoutCohort = students.filter((student) => {
+    return student.cohortId === null
+  })
+  const urlCohortId = useParams() // .id = "3"
+  const intCohortId = {
+    cohortId: parseInt(urlCohortId.id)
```
Comment (Auenc) [inline src/components/Cohorts/AddStudentPopUp/AddStudentPopUp.js:15]: Let's change `parseInt` to `Number` as `parseInt` has some weird behaviour if things aren't numbers!

File: src/components/Cohorts/AddStudentPopUp/AddStudentPopUp.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  const handleClick = (student) => {
+    client
+      .patch(`/users/${student.id}`, intCohortId)
+      .then(() => {
+        updateStudentsList()
+      })
+      .catch((err) => {
+        console.log('Error while patching user Cohort: ', err)
```
Comment (Auenc) [inline src/components/Cohorts/AddStudentPopUp/AddStudentPopUp.js:25]: Let's change this to a `console.error`

**Review by Auenc — CHANGES_REQUESTED** [review]

**Review by Auenc — APPROVED** [review]

**Pull request merged**